src/shared/codelens/codeLensUtils.ts

```diff
@@ -158,5 +152,207 @@ async function getAssociatedSamTemplate(
         return template
     }
 
-    throw new Error(`Unable to find a sam template associated with handler '${handlerName}' in ${documentUri.fsPath}.`)
+    throw new Error(`Cannot find a SAM template associated with handler '${handlerName}' in: ${documentUri.fsPath}.`)
+}
+
+export async function makePythonCodeLensProvider(
```

Comment:
Should this and the other language-specific functions in this file be moved to the language-specific CodeLensProvider files?

Reply:
There will likely be opportunity to DRY this up later. Having these functions live near each other meanwhile is a good reminder...